client/payment-methods/index.js

```diff
@@ -5,6 +5,7 @@
 import React from 'react';
 import { __ } from '@wordpress/i18n';
 import { Button, Card, CardBody } from '@wordpress/components';
+import { useState } from '@wordpress/element';
```

Comment:
(minor) it looks like there's already React in scope - is there a reason to prefer importing `useState` from `@wordpress/element` rather than `react`?

Reply:
Yeah, it is! The thing with `@wordpress/element` exposing some React methods is having a wrapper around React just in case React decides to change their API in the future, or if we need to introduce some changes to the API that React does not support. Currently I think it's almost a 1:1 equivalence, but this may change in the future. I don't remember where I read it... I've been searching but I can't find the source of my explanation, sorry! Probably in #gutenberg can give you more context.